code/modules/reagents/reagents/nanites.dm

```diff
@@ -92,52 +98,11 @@
 	if(..() && M.radiation)
 		return TRUE
 
-
 /datum/reagent/nanites/arad/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
 	if(..())
 		M.radiation = max(M.radiation - (5 + M.radiation * 0.10) * effect_multiplier, 0)
 
-
-/datum/reagent/nanites/implant_medics
-	name = "Implantoids"
-	id = "implant nanites"
-	description = "Microscopic construction robots programmed to repair implants."
-
-
-/datum/reagent/nanites/implant_medics/will_occur(mob/living/carbon/M, alien, var/location)
-	if(..() && ishuman(M))
-		var/mob/living/carbon/human/H = M
-		constant_metabolism = FALSE
-		metabolism = initial(metabolism)
-		for(var/obj/item/organ/organ in H.organs) //Grab the organ holding the implant.
-			if((organ.damage > 0) && BP_IS_ROBOTIC(organ)) //only robotic organs
-				return TRUE
-			if(istype(organ, /obj/item/organ/external))
-				var/obj/item/organ/external/E = organ
-				for(var/obj/item/implant/I in E.implants)
-					if(I.malfunction)
-						metabolism = 1
-						constant_metabolism = TRUE
-						return TRUE
-
-
-/datum/reagent/nanites/implant_medics/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
-	if(..() && ishuman(M))
-		var/mob/living/carbon/human/H = M
-		for(var/obj/item/organ/organ in H.organs) //Grab the organ holding the implant.
-			if(metabolism == 1 && istype(organ, /obj/item/organ/external)) // if metabolism == 1 then broken implant is found see implant_medics/will_occur()
-				var/obj/item/organ/external/E = organ
-				for(var/obj/item/implant/I in E.implants)
-					if(I.malfunction)
-						I.restore()
-						return
-			else if (istype(organ, /obj/item/organ/external) && organ.damage > 0 && BP_IS_ROBOTIC(organ))
-				organ.heal_damage((2 + organ.damage * 0.05)* effect_multiplier, (2 + organ.damage * 0.05)* effect_multiplier, 1, 1)
-				return
-			else if (istype(organ, /obj/item/organ/internal) && organ.damage > 0 && BP_IS_ROBOTIC(organ))
-				organ.heal_damage((2 + organ.damage * 0.05)* effect_multiplier)
-				return
-
```

Comment:
Removal comment needed

Reply:
Removal Comment was added at the top of the file

1. /* This file has been completely restructured for Occulus*/